src/snapshots/fuzz_crash/fuzz_crash_024.md

```diff
@@ -114,7 +114,9 @@ KwVar(7:1-7:4),LowerIdent(7:5-7:6),OpAssign(7:6-7:7),Int(7:8-7:9),EndOfFile(7:9-
 ~~~
 # FORMATTED
 ~~~roc
-module []
+module [
+	,
+]
```

Comment:
Why is this adding a comma?

Reply:
This is because we don't format malformed nodes but with my changes they are now considered to be multi-line.
So, in practice there is an item there, it is just not printed.